index.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   name: require('./package').name,
 
   // enable file-watching / live-reload
-  // isDevelopingAddon: () => true,
+  isDevelopingAddon: () => true,
```

Comment:
Do you plan to leave it as true or comment out before merging?

Reply:
I originally was, but lately I've been running out of file watchers, so I will undo this change